site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1372,6 +1373,25 @@ public function showGrading(
                 }
             }
             if (is_null($who_id) || $who_id == '') {
+                $message = "You've reached the ";
+                if ($to === 'prev') {
+                    $message .= " start";
+                }
+                else {
+                    $message .= " end";
+                }
+                $message .= " of";
+                if ($navigate_assigned_students_only !== "false") {
+                    $message .= " your assigned sections";
+                }
+                else {
+                    $message .= " the list";
+                }
```

Comment:
use ternary operator to condense some of this like $navigate_assigned_students_only !== "false" ?  $message .= " your assigned sections" : $message .= " the list";